codegen/examples/gen_mgmt.rs

```diff
@@ -3,61 +3,69 @@
+    "deviceprovisioningservices", // TODO #82 certificate_name used as parameter more than once
+    "dnc",                        // https://github.com/Azure/azure-rest-api-specs/pull/11578 two ControllerDetails types
+    "mixedreality",               // TODO #83 AccountKeyRegenerateRequest not generated
+    "netapp",                     // Ident "10minutely"
+    "powerplatform",              // https://github.com/Azure/azure-rest-api-specs/pull/11580 incorrect ref & duplicate Operations_List
+    "service-map",                // Ident "Ref:machine"
+    "servicefabric",              // https://github.com/Azure/azure-rest-api-specs/pull/11581 allOf mistakes and duplicate Operations_List
+    "web",                        // TODO #81 DataType::File
 ];
 
 const SKIP_SERVICE_TAGS: &[(&str, &str)] = &[
     ("azureactivedirectory", "package-preview-2020-07"),
     ("resources", "package-policy-2020-03"),
     ("resources", "package-policy-2020-09"), // SchemaNotFound { ref_key: RefKey { file_path: "../azure-rest-api-specs/specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json", name: "CloudError"
     ("recoveryservicesbackup", "package-2020-07"), // duplicate fn get_operation_status
+    ("recoveryservicesbackup", "package-2020-10"), // duplicate fn get_operation_status
     ("network", "package-2017-03-30-only"),  // SchemaNotFound 2017-09-01/network.json SubResource
     ("synapse", "package-2019-06-01-preview"), // TODO #80 path parameters
+    ("recoveryservicessiterecovery", "package-2016-08"), // duplicate package-2016-08 https://github.com/Azure/azure-rest-api-specs/pull/11287
+    ("mediaservices", "package-2019-05-preview"), // invalid unicode character of a dash instead of a hyphen https://github.com/Azure/azure-rest-api-specs/pull/11576
+    // datamigration, same error for all
+    // SchemaNotFound MigrateSqlServerSqlDbTask.json ValidationStatus, but may be buried
+    ("datamigration", "package-2018-07-15-preview"),
+    ("datamigration", "package-2018-04-19"),
+    ("datamigration", "package-2018-03-31-preview"),
+    ("datamigration", "package-2018-03-15-preview"),
+    ("datamigration", "package-2017-11-15-preview"),
+];
+
+// becuse of recursive types, some properties have to be boxed
+// https://github.com/ctaggart/autorust/issues/73
+const BOX_PROPERTIES: &[(&str, &str, &str)] = &[
+    // cost-management
+    ("../azure-rest-api-specs/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json", "ReportConfigFilter", "not"),
+    ("../azure-rest-api-specs/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json", "QueryFilter", "not"),
+    // network
+    ("../azure-rest-api-specs/specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/publicIpAddress.json", "PublicIPAddressPropertiesFormat", "ipConfiguration"),
```

Comment:
@bmc-msft, if you need to fix a different API version, you can make a pull request to add the entry here.